crates/katana/rpc/rpc/src/dev.rs

```diff
@@ -92,6 +97,35 @@ impl<EF: ExecutorFactory> DevApiServer for DevApi<EF> {
         Ok(())
     }
 
+    async fn account_balance(&self, address: String) -> Result<u128, Error> {
+        let account_address: ContractAddress = Felt::from_str(&address).unwrap().into();
+        let provider = self.backend.blockchain.provider();
+        let state = provider.latest().unwrap();
+        // let storage_slot =
+        //     get_storage_var_address("ERC20_balances", &[account_address.into()]).unwrap();
+        let storage_slot = get_fee_token_balance_base_storage_address(account_address);
+        let balance_felt = state
+            .storage(self.backend.chain_spec.fee_contracts.eth, storage_slot)
+            .unwrap()
+            .unwrap();
+        let balance: u128 = balance_felt.to_string().parse().unwrap();
+        Ok(balance)
+    }
```

Comment:
we should update this method to reflect to follow this 

https://github.com/0xSpaceShard/starknet-devnet-rs/blob/main/crates/starknet-devnet-server/src/api/http/endpoints/accounts.rs